app/shared/web3/interest-rate-to-apy.ts

```diff
@@ -0,0 +1,7 @@
+export function convertInterestRateToAPY(interestRatePerBlock: bigint, decimals: number, blocksPerDay: number): bigint {
```

Comment:
We have this file for such code
https://github.com/eonian-core/web/blob/main/app/finances/apy.tsx